handler/templates/handler.js

```diff
@@ -1,28 +1,29 @@
-import React from 'react';
-import {Resolver} from 'react-resolver';
+/** @flow */
+
+require('./styles.css');
+
+import React, {Component, PropTypes} from 'react';
+
+class <%= RouteName %> extends Component {
+
+  state = {
+    msg: 'Hello World'
+  };
```

Comment:
I don’t :heart: static property initializers being an expected thing for users to use by using this yet...